functions/controllers/applicants/index.js

```diff
@@ -0,0 +1,42 @@
+const functions = require("firebase-functions");
+const express = require("express");
+const cors = require("cors");
+const { jwtCheck, hasPermission } = require("../../utils/middleware");
+const { queryDocument } = require("../../utils/database");
+
+const applicant = express();
+applicant.disable("x-powered-by");
+applicant.use(express.json());
+
+const auth0Config = functions.config().auth;
+const corsConfig = auth0Config ? auth0Config.cors : "";
+
+const corsOptions = {
+  origin: corsConfig,
+  optionsSuccessStatus: 200,
+};
+
+applicant.use(cors(corsOptions));
+
+applicant.get("/applicant/:id/", jwtCheck, hasPermission, async (req, res) => {
```

Comment:
I believe you need to specify which permission middleware for `hasPermission`. In this case I think it would be `hasReadAdmin` which needs to be imported from middleware.js.